Snowflake.Data/Core/Authenticator/ExternalBrowserAuthenticator.cs

```diff
@@ -61,29 +63,21 @@ async Task IAuthenticator.AuthenticateAsync(CancellationToken cancellationToken)
                 authenticatorRestResponse.FilterFailedResponse();
 
                 var idpUrl = authenticatorRestResponse.data.ssoUrl;
-                proofKey = authenticatorRestResponse.data.proofKey;
+                _proofKey = authenticatorRestResponse.data.proofKey;
 
                 logger.Debug("Open browser");
                 StartBrowser(idpUrl);
 
                 logger.Debug("Get the redirect SAML request");
-                var context = await httpListener.GetContextAsync().ConfigureAwait(false);
-                var request = context.Request;
-                samlResponseToken = ValidateAndExtractToken(request);
```

Comment:
Why we removed samlResponseToken and related response code? I am not seeing any new code related to that.